### DISCUSSION TITLE
Can I use Moose to calculate resonance modes?

### DISCUSSION THREAD

**Discussion by ghmechamania** [idaholab/moose#28280] [Q&A General]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

Hi all,
I'm new to FEM and I'm investigating if I can use Moose for my new project.
For this project I would like to calculate the resonance frequencies and modes of 2D an 3D objects.
This to be able to synthesize the sound an object would make when hitting it at some position.

So can I use Moose to calculate these modes? If so, can you give me some hints how to tackle this?
Thanks.

**Comment by maxnezdyur** [comment]
This may be what you are looking for https://mooseframework.inl.gov/modules/solid_mechanics/1d_elastic_waves.html
There are two ways to do this, but the modal analysis way would get you more modes at once without needing to sweep.
